cluster/src/main/java/org/apache/iotdb/cluster/client/IClientPool.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.iotdb.cluster.client;
+
+import org.apache.iotdb.cluster.rpc.thrift.Node;
+import org.apache.iotdb.cluster.rpc.thrift.RaftService;
+
+import java.io.IOException;
+
+public interface IClientPool {
```

Comment:
We'd better unify the context. Either use `clientPool` or `clientManager`. Otherwise it would seem strange to have `ClientManager` inherit `IClientPool`

Reply:
Yes, the I named it `IClientManager` at first and change to `IClientPool` as I think the interfaces are more like a pool actions. Change to `IClientManager` to make it consistency.